next.config.js

```diff
@@ -901,7 +901,7 @@ module.exports = withFonts(
       trailingSlash: true,
     },
     env: {
-      FATHOM_ID: 'EPNTIXUM',
+      FATHOM_ID: 'FOEMPXUV',
```

Comment:
why is this change needed? I'd be worried about losing historic data for the docs

Reply:
We already have a significant gap in the historical analytics due to this breakage.  Since you can't rename a site on Fathom I thought it would be good to have a dashboard with the appropriate `docs.stacks.co` name, no reason we can't keep the old dashboard around for historical reasons though.

Reply:
ah makes sense